app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def self.current_user
+    User.first
+  end
```

Comment:
- Your app works fine, even though personally I would prefer this way of creating the current user on the application controller:
```ruby
# app/controllers/application_controller.rb
...

def current_user
  User.first
end
```

Then for example in the posts_controller you can get the current user like this: 
```ruby
def new
      @post = Post.new
    end

    def create
      @post = @current_user.posts.new(post_params)

      respond_to do |format|
        if @post.save
          format.html { redirect_to @post }
        else
          format.html { render :new }
        end
      end
    end
```
Your project would look more cleaner and better looking. But still your application satisfies all the requirements so I will go ahead and approve it. Good job :clap: